portfolio/src/main/java/com/google/sps/servlets/CheckLogin.java

```diff
@@ -19,9 +19,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     UserService userService = UserServiceFactory.getUserService();
     if (userService.isUserLoggedIn()) {
-      response.getWriter().println("1");
+      response.getWriter().println("1"); // "1" shows that a user is logged into the website.
```

Comment:
Since you're just sending a string, why not use something more clear like "logged in"?  I think it's not visible to the user and it makes it easier to debug later.  I'm not familiar with how the library works so if there's a good reason to leave it, just let me know.